modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -1,26 +1,28 @@
 // Module included in the following assemblies:
 //
-// * networking/ingress-controller-configuration.adoc
+// * networking/ingress-operator.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-autoscaling-ingress-controller_{context}"]
 = Autoscaling an Ingress Controller
 
-Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+You can automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements, such as the requirement to increase throughput.
+
+The following procedure provides an example for scaling up the default Ingress Controller.
 
 .Prerequisites
-. You have the OpenShift CLI (`oc`) installed.
-. You have access to an {product-title} cluster as a user with the `cluster-admin` role.
-. You have the Custom Metrics Autoscaler Operator installed.
-. You are in the `openshift-ingress-operator` project namespace.
+
+* You have the {oc-first} installed.
+* You have access to an {product-title} cluster as a user with the `cluster-admin` role.
+* You installed the Custom Metrics Autoscaler Operator.
 
 .Procedure
 
 . Create a service account to authenticate with Thanos by running the following command:
 +
 [source,terminal]
 ----
-$ oc create serviceaccount thanos && oc describe serviceaccount thanos
```

Comment:
Before create the service account , there should be two steps as below
1. Do all operations in the openshift-ingress-operator namespace:
`$ oc project openshift-ingress-operator`

2. Enable OpenShift monitoring for user-defined projects
Please refer to the [documentation](https://docs.openshift.com/container-platform/4.9/monitoring/enabling-monitoring-for-user-defined-projects.html), or just apply the following ConfigMap:
```
$ oc apply -f - <<EOF
apiVersion: v1
kind: ConfigMap
metadata:
  name: cluster-monitoring-config
  namespace: openshift-monitoring
data:
  config.yaml: |
    enableUserWorkload: true
EOF
```

Reply:
Thanks. I added a line to the Prerequisites section.

Reply:
Thanks, Grant. I'll hold off until both of you approve the changes.

Reply:
@gcs278 Yes, it sounds great. Currently the following command needs the namespace openshift-ingress-operator
 oc adm policy add-role-to-user thanos-metrics-reader -z thanos --role-namespace=openshift-ingress-operator
 
 CC: @dfitzmau

Reply:
I tested it on 4.16 and 4.17, the both were failed. Same error logs: "error parsing prometheus metadata: no bearer token provided". It is strange I have created the secret manually.